packages/vite-native-client/src/client.ts

```diff
@@ -116,9 +116,9 @@ function setupWebSocket(
 
     notifyListeners('vite:ws:disconnect', { webSocket: socket })
 
-    console.log(`[vite] server connection lost. polling for restart...`)
+    console.info(`[vite] server connection lost. polling for restart...`)
     waitForSuccessfulPing(protocol, hostAndPath).then(() => {
-      console.log('shuld reload')
+      console.info('shuld reload')
```

Comment:
```suggestion
      console.info('should reload')
```

just a small typo :)